tutorials/incorporating_oli_calculations.ipynb

```diff
@@ -234,43 +296,85 @@
    "outputs": [],
    "source": [
     "if credential_manager:\n",
+    "    # OLIApi is used as a context manager\n",
     "    with OLIApi(credential_manager) as oliapi:\n",
     "        # create a new DBS file\n",
+    "        # alternative thermo_frameworks and databanks are available.\n",
+    "        \n",
+    "        # use keep_file = True to save a DBS file ID on the Cloud\n",
+    "        # for use in more than one session\n",
     "        dbs_file_id = oliapi.get_dbs_file_id(\n",
-    "            chemistry_source=source_water[\"components\"],\n",
+    "            inflows=inflows,\n",
+    "            thermo_framework=\"MSE (H3O+ ion)\",\n",
     "            phases=[\"liquid1\", \"solid\"],\n",
+    "            model_name=\"test\",\n",
+    "            keep_file=true,\n",
     "        )\n",
+    "        # NOTE: liquid2 (non-aqueous) and vapor phases are available but not yet fully supported.\n",
+    "                \n",
+    "        # if a DBS file has been retained from a previous session,\n",
+    "        # its flash history and chemistry information can be summarized.\n",
+    "        file_summary = oliapi.get_dbs_file_summary(dbs_file_id)\n",
     "        \n",
-    "        # create water analysis inputs\n",
-    "        water_analysis_input = f.build_flash_calculation_input(\n",
-    "            \"wateranalysis\",\n",
-    "            state_vars,\n",
+    "        # save chemistry information\n",
+    "        chemistry_info = file_summary[\"chemistry_info\"]\n",
+    "        write_output(chemistry_info[\"result\"], \"chemistry_info\")\n",
+    "        \n",
+    "        # create water analysis input\n",
+    "        # Water Analysis uses true species for inputs, i.e., IONS.\n",
+    "        json_input = flash.configure_water_analysis(\n",
+    "            inflows,\n",
+    "            reconciliation=\"ReconcilePh\",\n",
+    "            ph=5,\n",
+    "            allow_solids=True,\n",
+    "            file_name=\"water_analysis_inputs\",\n",
     "        )\n",
-    "    \n",
-    "        # create water analysis case\n",
-    "        water_analysis_base_case = f.run_flash(\n",
+    "        # run Water Analysis flash calculation survey as specified        \n",
+    "        stream_output = flash.run_flash(\n",
```

Comment:
Once this cell finishes running, I get a prompt for entering y/n but I have no information about what I am saying yes or no to. We should change this so that what the user is being prompted for is clear. The same goes in the following cell, which has another separate issue that I'll comment on.

Reply:
The message (WaterTAP will delete 0 DBS files) is in the logger, but if it's more clear to have a message in line with the prompt then that can be done